pages/common/hugo.md

```diff
@@ -0,0 +1,31 @@
+# hugo
+
+> Hugo is a Fast and Flexible Static Site Generator.
+
+- Create a new hugo site:
+
+`hugo new site {{path/to/site}}`
+
+- Create a new hugo theme (you may also download one from [https://themes.gohugo.io/]()):
+
+`hugo new theme {{theme_name}}`
+
+- Create a new page:
+
+`hugo new {{section_name}}/{{filename}}{{.ext}}`
+
+- Build your site to the `./public/` directory:
+
+`hugo`
+
+- Build your site to a different destination:
```

Comment:
We prefer not to have example descriptions use relative references, i.e. rely on context from previous examples. In this case, instead of "to a different destination", it would be preferable to say something like "to a given directory".